types/n3/index.d.ts

```diff
@@ -191,9 +191,8 @@ export type PrefixCallback = (prefix: string, prefixNode: RDF.NamedNode) => void
 
 export class Parser<Q extends BaseQuad = Quad> {
     constructor(options?: ParserOptions);
-    parse(input: string): Q[];
-    parse(input: string, callback: null | undefined, prefixCallback: PrefixCallback): Q[];
-    parse(input: string, callback: ParseCallback<Q>, prefixCallback?: PrefixCallback): void;
+    parse(input: string, callback?: null, prefixCallback?: PrefixCallback): Q[];
```

Comment:
@mielvds can you add a test case where `undefined` is explicitly provided as a parameter?

Reply:
And I think we should keep the explicit `undefined` given the introduction of `--exactOptionalPropertyTypes` in TS 4.4 (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-4.html)

Reply:
I removed the explicit `undefined` because the DefinitelyTyped tests were complaining: https://github.com/DefinitelyTyped/DefinitelyTyped/commit/c01e427dde51eaf526ed62ecc7c839119c2c551b/checks?check_suite_id=13299824905#step6:130. I added the tests

Reply:
Ah - my bad; it must only apply to properties on interfaces, rather than all optional types - thanks for adding the tests.

Reply:
So should I still make any changes or is it ok?

Reply:
It is ok